templates/default/tags/html/example.erb

```diff
@@ -1,9 +1,9 @@
 <% if object.has_tag?(:example) %>
   <div class="examples">
-    <p class="tag_title">Examples:</p>
+    <h4 class="tag_title">Examples:</h4>
```

Comment:
One requirement we had was to not have paragraph sections for examples. We set these to h4 and h5 headers so I think this is reasonable as they are supposed to be sections.